src/main/java/de/tum/in/www1/artemis/service/metis/conversation/ChannelService.java

```diff
@@ -451,6 +448,12 @@ private Channel updateChannelName(Channel channel, String newChannelName) {
         }
     }
 
+    /**
+     * Creates a channel object with some default properties
```

Comment:
just a suggestion: directly specify in the javadoc that it is a public channel but not an announcement channel and the channel ist not archived instead of writing "some default properties"  which requires me to read the code to understand what is actually set.